### PR TITLE
home work 16 functions

### PR DESCRIPTION
**Pull request by Oleksii1985**

### CODE REVIEW

File: lesson_8/funstions.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+
+function average(nums) {
+  result = [];
+  for (let i = 0; i < nums.length; i++) {
+    if (Number(nums[i])) {
+      result.push(nums[i]);
+    }
+  }
+  return result.reduce((a, b) => (a + b)) / result.length;
```
Comment (web-scavenger) [inline lesson_8/funstions.js]: dont forget initial argument
```suggestion
  return result.reduce((a, b) => (a + b), 0) / result.length;
```

File: lesson_8/funstions.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+/**
+ * @param {array <any>} nums 
+ * 
+ * @returns array <number>
+ */
+
+
+function average(nums) {
+  result = [];
```
Comment (web-scavenger) [inline lesson_8/funstions.js]: Why do you still don't use let or const to create variables??

File: lesson_8/funstions.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  for (let i = 0; i < nums.length; i++) {
+    if (Number(nums[i])) {
```
Comment (web-scavenger) [inline lesson_8/funstions.js:15]: it will be better to use `typeof` for check

File: lesson_8/funstions.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      result = x / y * 100;
+      return `${x} % ${y} = ${result}`;
+
+    case '^':
+      result = x ** y;
+      return `${x} ^ ${y} = ${result}`;
+
+    default:
+      return 'Invalid operator';
+  }
```
Comment (web-scavenger) [inline lesson_8/funstions.js:72]: Why didn't use functions for this task?

File: lesson_8/funstions.js (hunk 2 above)
Comment (Oleksii1985) [reply lesson_8/funstions.js]: because of PYTHON)))))

**Pull request merged**